docker-compose.yml

```diff
@@ -1,25 +1,44 @@
-# docker-compose for socorro-collector that creates a minimal collector.
-#
 # Note: Requires docker 1.10.0+ and docker-compose 1.6.0+.
 version: "2"
 services:
   db:
     image: postgres:9.4
 
   web:
-    build: 
-      context: .
+    image: local/standup_dev
     ports:
       - "8000:8000"
     volumes:
       - .:/app
     links:
       - db
     environment:
-      - PYTHONDONTWRITEBYTECODE=1
       - DATABASE_URL=postgres://postgres@db/postgres
       - DEBUG=True
       - ALLOWED_HOSTS=localhost,127.0.0.1,
       - SECRET_KEY=59114b6a-2858-4caf-8878-482a24ee9542
+
+  prod:
+    image: local/standup_base
+    ports:
+      - "8000:8000"
+    volumes:
+      - .:/app
+    links:
+      - db
+    environment:
+      - DATABASE_URL=postgres://postgres@db/postgres
+      - DEBUG=False
+      - ALLOWED_HOSTS=localhost,127.0.0.1,
+      - SECRET_KEY=59114b6a-2858-4caf-8878-482a24ee9542
```

Comment:
Not planning on running this as the prod environment. In fact docker-compose won't be used in prod. This is meant as a way to try running the site locally but in a prod-like environment. It should be documented as such though. And it might be better to use a non-official-looking key here: something like `ITSASEKRIT` that is obviously a placeholder?